ext/opcache/Optimizer/pass1_5.c

```diff
@@ -309,6 +307,11 @@ void zend_optimizer_pass1(zend_op_array *op_array, zend_optimizer_ctx *ctx)
 						}
 					}
 				} else if (op_array->scope &&
+					ZEND_OP1_TYPE(opline) == IS_UNUSED &&
+					(opline->op1.num & ~ZEND_FETCH_CLASS_NO_AUTOLOAD) == ZEND_FETCH_CLASS_SELF) {
```

Comment:
This check and the already existing one in L318 might currently be dead, because constant fetch is generated with ZEND_FETCH_CLASS_EXCEPTION.